torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -1300,3 +1305,138 @@ AOTITorchError aoti_torch_cpu__weight_int4pack_mm_cpu_tensor(
     *ret0 = new_tensor_handle(std::move(tmp_result));
   });
 }
+
+class StableIValueConverter : public c10::OperatorKernel {
```

Comment:
Why is this a `StableIValueConverter`?

Reply:
This sounds more like a "StableBoxedKernel"

Reply:
This is the boxed struct to contain the functor that liaisons between IValues in torch::jit::stack and StableIValue in the user kernel

Reply:
Oh, but the BoxedKernel is the thing that consumes this, no?

Reply:
My confusion was: this thing seems like a "Kernel", but it's named "StableIValueConverter"

Reply:
Renaming to StableIValueBoxedKernel to be super clear cuz it is indeed a kernel that just converts ivalues